test/functionalTest/cases/0876_entity_dates/entity_dates_overridden_by_user.test

```diff
@@ -38,8 +38,8 @@ brokerStart CB
 # 03. GET /v2/entities/E1?attrs=dateModified,dateCreated and get user defined dates
 # 04. GET /v2/entities?q=dateCreated>2049-01-01 and get nothing [should be E1]
 # 05. GET /v2/entities?q=dateModified>2049-01-01 and get nothing [should be E1]
-# 06. GET /v2/entities?q=dateCreated<2049-01-01 and get E1 [should get nothing]
```

Comment:
Diff is pretty noisy in this file. I'd suggest to have a look to the final file and ensure it correct regarding what is told in section "User attributes or metadata matching builtin name" of ngsiv2_implementation_notes.md file.

Reply:
NTC